hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/libhdfspp/CMakeLists-gtest.txt.in

```diff
@@ -1,3 +1,6 @@
+# Copyright 2008, Google Inc.
+# All rights reserved.
+
```

Comment:
We don't have to add the header here because the header is already included in ./licenses/LICENSE-gtest.txt and ./licenses-binary/LICENSE-gtest.txt.

Reply:
Done. Thanks.